src/bundle/Resources/public/scss/_date-time-picker.scss

```diff
@@ -19,8 +19,20 @@
         width: 100%;
     }
 
-    .ibexa-input-text-wrapper__actions {
-        top: calculateRem(24px);
+    .ibexa-input-text-wrapper {
+        &__actions {
+            top: calculateRem(24px);
+        }
+
+        &__action-btn--calendar-icon {
+            &:hover {
+                cursor: pointer;
+
+                .ibexa-icon {
+                    fill: $ibexa-color-primary;
```

Comment:
Should we add the `animation` property? Maybe this should be a modified ghost button like in the case of other action buttons from `src/bundle/Resources/views/themes/admin/ui/component/input_text.html.twig`? 🤔